question-3.md

```diff
@@ -3,4 +3,15 @@ layout: question-include
 title: Question 3
 number: 3
 type: radio
----
\ No newline at end of file
+---
+
+**Examples of health information rights::**
+
+- [HIPAA Privacy Rule ](https://www.hhs.gov/hipaa/for-individuals/guidance-materials-for-consumers/index.html?language=es)<br>
```

Comment:
This copy works pretty well, but in general, I think it would be more helpful to lead with the explanation. Most people aren't to click or mentally process these official names. But they will understand "Federal law recognizes your rights over your health information and set rules and limits on who can see and receive it." So maybe reversing them and following with the link in parentheses?